tests/queries/0_stateless/03013_test_part_level_is_reset_attach_from_disk_mt.sql

```diff
@@ -0,0 +1,19 @@
+-- {echoOn}
```

Comment:
```suggestion
SET alter_sync = 2;
-- {echoOn}
```
After going over second time, I think this can be useful. In some wierd cases this test might be flaky if the server slow to optimize/detach/attach parts. By explicitly waiting on these operations to finish the test can be more robuts.

Reply:
Done